### ISSUE TITLE
Unable to read Json Payload from Request Body using "AuthService"

### ISSUE THREAD

**Issue by kasrajag** [datawire/ambassador#1014]
Hello,

We are trying to do some validation based on Request body payload data using Ambassador "AuthService".
But my service is unable to read payload data, its showing NULL
I dont see any attribute or config value for "Authservice" related to payload
Any inputs on this ?

Thanks in advance!

**Comment by enriched** [comment]
The request that hits the AuthService has the body removed [per the documentation](https://www.getambassador.io/reference/services/auth-service#the-request) I have not seen anything in the documentation about how to override that behavior.

**Comment by richarddli** [comment]
This is addressed in 0.50-EA7 (and will GA in 0.50). Per the documentation linked above:

```
allow_request_body (optional) will pass the full body of the request to the auth service for use cases such as computing an HMAC or request signature.
```
